qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -161,9 +161,10 @@ def setUp(self):
 
             # In case some test messed with auth caps, reset them
             for client_id in client_mount_ids:
-                cmd = ['auth', 'caps', f'client.{client_id}', 'mon','allow r',
-                       'osd', f'allow rw pool={self.fs.get_data_pool_name()}',
-                       'mds', 'allow']
+                cmd = ['auth', 'caps', f'client.{client_id}',
+                       'mds', f'allow rwp fsname={self.fs.name}',
```

Comment:
@trociny Looks like we have a couple of issues here:

This line is failing to be parsed resulting and results in the following error
```                                              
2023-03-01T04:31:47.882 INFO:teuthology.orchestra.run.smithi008.stderr:Error EINVAL: mds capability parse failed, stopped at 'fsname=cephfs' of 'allow rwp fsname=cephfs'                                         
2023-03-01T04:31:47.904 DEBUG:teuthology.orchestra.run:got remote process result: 22
```

Failed job: https://pulpito.ceph.com/vshankar-2023-03-01_02:28:49-fs-wip-vshankar-testing-20230228.105516-testing-default-smithi/7190048/

I didn't debug this in detail as I'm on PTO today.

Another thing is we need to include the 's' flag (after rwp) for the user to be able to create and delete snapshots. See: https://docs.ceph.com/en/latest/cephfs/client-auth/#snapshot-restriction-the-s-flag

Reply:
As I understand the issue is that we are running this command against nautilus cluster and it does not support the `fsname` keyword yet.  Actually, I had to fix the osd caps only, and `'allow rw tag cephfs data={self.fs.name}'` seems to work on nautilus too. The mds and mon changes were just "improvement" to have it more strict. I will try just reverting them back and see if it helps.

As for the missing 's' flag, in the original version we just had `'mds', 'allow'`, which I suppose includes all flags, right? Then I will just revert to that version.

Reply:
> As I understand the issue is that we are running this command against nautilus cluster and it does not support the `fsname` keyword yet. Actually, I had to fix the osd caps only, and `'allow rw tag cephfs data={self.fs.name}'` seems to work on nautilus too. The mds and mon changes were just "improvement" to have it more strict. I will try just reverting them back and see if it helps.
> 
> As for the missing 's' flag, in the original version we just had `'mds', 'allow'`, which I suppose includes all flags, right? Then I will just revert to that version.

Right.